public/pages/Dashboard/components/FindingsDashboard/FindingFlyout.js

```diff
@@ -93,17 +101,32 @@ export default class FindingFlyout extends Component {
 
     const flyout = (
       <EuiFlyout
-        type={isAlertsFlyout ? 'overlay' : 'push'}
         onClose={this.closeFlyout}
-        ownFocus={false}
+        ownFocus={true}
         hideCloseButton={true}
         side={'right'}
         size={'m'}
       >
         <EuiFlyoutHeader hasBorder>
-          <EuiTitle size={'m'}>
-            <h2 id={findingId || `temp_finding_${docId}`}>Document finding</h2>
-          </EuiTitle>
+          <EuiFlexGroup justifyContent="flexStart" alignItems="center">
+            <EuiFlexItem>
+              <EuiFlexGroup alignItems="center">
```

Comment:
Similarly here, we should simplify this by not having the flex group

Reply:
done